html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <div>
+    Email
+    <br />
+    <label for="email">
```

Comment:
The label tag should wrap the title for that specific form element, not the form element itself. For example:
< label for="email-address" > Email </ label >